bin/common.bash

```diff
@@ -399,9 +399,13 @@ validate_sops_config() {
         exit 1
     fi
 
-    # Force all pgp entries to be a string and to contain "null" if unset in a creation rule
-    fingerprints=$(yq4 'with(.creation_rules[]; .pgp = (.pgp // "null" | to_yaml style="")) | [.creation_rules[].pgp] | join(",")' "${sops_config}")
-    if ! [[ "${fingerprints}" =~ ^[A-Z0-9,' ']+$ ]]; then
+    # Compares the keyring with pgp fingerprints to see if pgp has anything keyring does not have.
+    keyring=$(gpg --list-keys  | grep pub -A 1 | grep -v pub | sed -n 'n;p' | xargs)
+    creation_pgp=$(yq4 '.creation_rules[].pgp // null' "${sops_config}")
```

Comment:
```suggestion
    creation_pgp=$(yq4 '[.creation_rules[].pgp // null | split(",") | .[]] | unique | join(" ")' "${sops_config}")
```
Since else keys in multiple rules will pass unconditionally.